spec/integration/action/executeSqlInjection.test.js

```diff
@@ -41,4 +145,15 @@ describe('GeneralSqlQuery Action test', () => {
     await generalSqlQuery.process.call(emitter, msgWithError, cfg);
     expect(emitter.emit.calledWith('error')).to.be.equal(true);
   });
+
+  it('should be deadlock', async () => {
+    // for i in {1..3}; do psql elasticio_testdb postgres -c "select f_test('blah')"; done
```

Comment:
Should it be deleted?

Reply:
I think, this note (shell command) will be useful for understanding the mechanism of the test